src/components/Recommend/Recommend.js

```diff
@@ -0,0 +1,179 @@
+import React, { useEffect, useState } from 'react'
+import RNPickerSelect from 'react-native-picker-select'
+import { Text, View } from 'react-native'
+import { TouchableNativeFeedback } from 'react-native-gesture-handler'
+import PropTypes from 'prop-types'
+import { getPlaylistsByOffering, getVideosByPlaylist } from '../../api/playlists'
+import { STACK_SCREENS } from '../../containers/CTNavigationContainer/index'
+import { FILE_SERVER_BASE_URL } from '../../constants'
+import styles from './Recommend.style'
+
+/**
+ * Class for recccomend video of corresbonding course
+ * @param {Object} navigation navigation ovject for CTNavigator
+ * @param {string} courseId courseId used for reccomend video
+ * @param {boolean} mode mode for latestvideo, false mean progress tracking.
+ * @returns
+ */
+const Recommend = ({ navigation, courseId, mode }) => {
```

Comment:
Also, I just realized that Reccommend is spelled wrong. It should be Recommend. Can you update this.